section-09-differential-testing/model-package/differential_tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from regression_model.config.core import config
+from regression_model.processing.data_manager import load_dataset
+from regression_model import __version__ as model_version
+import os
+
+
+@pytest.fixture
```

Comment:
Adding a short comment to explain what we will use these functions for would be helpful.

Reply:
done